src/workloads/execution/SinusoidalReadWrites.yml

```diff
@@ -75,37 +87,56 @@ ActorTemplates:
             ThrowOnFailure: false
 Actors:
 - Name: Setup
-  Type: Loader
+  Type: AdminCommand
   Threads: 1
   Phases:
     OnlyActiveInPhases:
       Active: [0]
       NopInPhasesUpTo: *NumPhases
       PhaseConfig:
-        # Have 500 documents ready at start
         Repeat: 1
-        BatchSize: 100
+        Database: admin
+        Operations:
+        - OperationName: AdminCommand
+          OperationCommand:
+            setParameter: 1
+            ttlMonitorSleepSecs: 1
```

Comment:
Does this mean we'll be running the TTL monitor every 1 second?

Reply:
This means the TTL monitor will sleep 1 second after a pass.

A pass can continue indefinitely until there are no documents left to delete after the most recent subpass.

A subpass takes a snapshot of the TTLCollections, and continuously iterates over each index, deleting up to n-documents, until there are no more documents to delete on said index. A subpass is limited by a timer, to ensure that if new TTLIndexes are added, they won't be starved of deletes during the remainder of the pass.